vector-im/element-x-android
# Render send state and read receipts

## Type of change


## Content


Render send status, for the messages sent by the user.
Render read receipt, behind a disabled-by-default feature flag for now, since there are still some issue SDK side.

## Motivation and context


Closes #1808 

## Screenshots / GIFs


See recorded screenshots.

## Tests



- Go to a room and send a message to observe the send state icon
- Enable the new flag for read receipt and go to a room to see the read receipt.
- Click on the read receipt to see the bottom sheet with all the read receipts.
- On the bottom sheet, items are clickable and open the room member detail page. 

## Tested devices

- OS version(s):

## Checklist
